modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -12,7 +12,7 @@ In addition to the global pod security admission control configuration, a contro
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. You can enable pod security admission synchronization on other namespaces as necessary.
+Most namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. Namespaces that are prefixed with `openshift-` and contain an Operator have pod security admission synchronization enabled when a cluster service version (CSV) is created. You can enable pod security admission synchronization on other namespaces as necessary.
```

Comment:
again i think `openshift-operators` is the only exception to this rule (and at this point i'm not even sure we should have it as an exception, since we explicitly label anyway, so syncing isn't needed).

Reply:
> Namespaces that are prefixed with `openshift-` and contain an Operator have pod security admission synchronization enabled when a cluster service version (CSV) is created.

Non-cluster-payload namespaces that are prefixed with `openshift-` and contain an Operator will have pod security admission synchronization enabled when a cluster service version (CSV) is created.